dbt/README.md

```diff
@@ -128,6 +128,7 @@ Run the following commands in this directory:
 ```
 python3 -m venv venv
 source venv/bin/activate
+pip install -U pip
```

Comment:
Older versions of pip are subject to [a known bug](https://github.com/pypa/pip/issues/10851) that prevents packages from installing correctly:

```
$ pip install -r requirements.txt
...
ERROR: Exception:
Traceback (most recent call last):
  File "/home/jecochr/code/data-architecture/dbt/new-venv/lib/python3.10/site-packages/pip/_internal/cli/base_command.py", line 165, in exc_logging_wrapper
    status = run_func(*args)
  File "/home/jecochr/code/data-architecture/dbt/new-venv/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 205, in wrapper
    return func(self, options, args)
  File "/home/jecochr/code/data-architecture/dbt/new-venv/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 389, in run
    to_install = resolver.get_installation_order(requirement_set)
  File "/home/jecochr/code/data-architecture/dbt/new-venv/lib/python3.10/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 188, in get_installation_order
    weights = get_topological_weights(
  File "/home/jecochr/code/data-architecture/dbt/new-venv/lib/python3.10/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 276, in get_topological_weights
    assert len(weights) == expected_node_count
AssertionError
```

Making sure pip is up to date with the latest version in the virtualenv resolves this bug.